CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Bug Fixes
 
 1. [15731](https://github.com/influxdata/influxdb/pull/15731): Ensure array cursor iterator stats accumulate all cursor stats
+1. [15769](https://github.com/influxdata/influxdb/pull/15769): Alphabetized bucket list dropdown in delete with predicate
```

Comment:
do we want to add a line in the changelog for a feature that's not released?

Reply:
totally right